docs/client-configuration.md

```diff
@@ -13,7 +13,7 @@
 | socket.keepAlive             | `true`                                   | Toggle [`keep-alive`](https://nodejs.org/api/net.html#socketsetkeepaliveenable-initialdelay) functionality                                                                                                                                          |
 | socket.keepAliveInitialDelay | `5000`                                   | If set to a positive number, it sets the initial delay before the first keepalive probe is sent on an idle socket                                                                                                                                   |
 | socket.tls                   |                                          | See explanation and examples [below](#TLS)                                                                                                                                                                                                          |
-| socket.reconnectStrategy     | `retries => Math.min(retries * 50, 500)` | A function containing the [Reconnect Strategy](#reconnect-strategy) logic                                                                                                                                                                           |
+| socket.reconnectStrategy     | `((retries^2) * 50 ms) + 0-200 ms`       | A function containing the [Reconnect Strategy](#reconnect-strategy) logic                                                                                                                                                                           |
```

Comment:
I went with a plain english explanation of the strategy in [38eb603](https://github.com/redis/node-redis/pull/2736/commits/38eb60306b6974c5950432abd0ee542d5980748e), rather than try to figure out how to pseudo-code this in a legible way.